generation/WinSDK/documentationMappings.rsp

```diff
@@ -13781,6 +13781,7 @@ DXGI_ADAPTER_FLAG3=[Documentation("https://learn.microsoft.com/windows/win32/api
 DXGI_ALPHA_MODE=[Documentation("https://learn.microsoft.com/windows/win32/api/dxgi1_2/ne-dxgi1_2-dxgi_alpha_mode")]
 DXGI_COLOR_SPACE_TYPE=[Documentation("https://learn.microsoft.com/windows/win32/api/dxgicommon/ne-dxgicommon-dxgi_color_space_type")]
 DXGI_COMPUTE_PREEMPTION_GRANULARITY=[Documentation("https://learn.microsoft.com/windows/win32/api/dxgi1_2/ne-dxgi1_2-dxgi_compute_preemption_granularity")]
+DXGI_CREATE_FACTORY_FLAGS=[Documentation("https://learn.microsoft.com/windows/win32/api/dxgi/nf-dxgi-createdxgifactory")]
```

Comment:
@mikebattista any thought about linking the relevant function page if there's no type documentation available at all? Or should I revert all these additions to `documentationMappings.rsp`?

Reply:
I'd delete them. Bindings may generate incorrect summaries/comments.